plugins/manipulator.cpp

```diff
@@ -2291,6 +2301,9 @@ DFhackCExport command_result plugin_enable(color_ostream &out, bool enable)
             return CR_FAILURE;
 
         is_enabled = enable;
+
+        if (is_enabled)
```

Comment:
Is there any harm in leaving alchemist enabled even if manipulator is later disabled? it seems disabling it could lead to "dueling plugins" if some other plugin wants to keep it enabled for some reason.

Reply:
I think it would make sense to leave enabled (although perhaps from autohauler, per my other comment)

Reply:
thanks! moved to autohauler and left enabled